README.md

```diff
@@ -64,6 +64,7 @@ The following table shows which versions of the SDK are compatible with the tool
 | 100.12.0 | 100.12.0 |
 | 100.13.0 | 100.13.0 |
 | 100.14.0 | 100.14.0 |
+| 100.15.0 | 100.15.0 |
 
 ## Resources
 
```

Comment:
I wonder do we have to update the licence text below to say 2022 instead of 2020? I don't really know how licences work..
https://github.com/Esri/arcgis-runtime-toolkit-android/blob/9e7d89ad0ef3769ab9a1701091b05b3450b9baa9/README.md?plain=1#L86

Reply:
good catch, updated